.github/workflows/ui-test-for-commit.yml

```diff
@@ -22,7 +22,7 @@ jobs:
           key: ${{ runner.os }}-playwright-${{ env.PLAYWRIGHT_VERSION }}
       - name: Install
         run:
-          npm ci --prefer-offline --no-audit --no-fund
```

Comment:
The whole idea of using npm ci in the workflow  is not to use npm install. coz you need clean install to detect any mismatch or incompatible versions as it install directly from the lock.json file and compare with package.json. adding npm run bootstrap is basically using npm install (here using both ci and install) and this will update the lock file and this is not desirable in a workflow automated environment. I have a feeling this is a cache issue why not just remove the flag --prefer-offline and try. Since this flag is to speedup the ci command using cache.